rust/ommx/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "ommx"
-version = "0.1.0"
+version = "0.1.1"
```

```suggestion
version = "0.2.0"
```

これはnon-exhaustiveでないstructにpubなfiledを追加することになるので破壊的な変更です